vladiate/vlad.py

```diff
@@ -14,6 +14,7 @@ def __init__(
         default_validator=EmptyValidator,
         delimiter=None,
         ignore_missing_validators=False,
+        file_validation_threshold=10.0,
```

Comment:
I think the default should be to maintain the current behavior:

```suggestion
        file_validation_threshold=None,
```

We should document this new parameter as well here: https://github.com/di/vladiate#running-vlads-programatically